<ui version="4.0" >
 <class>partListDialog</class>
 <widget class="QDialog" name="partListDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parts</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/partlist.png</normaloff>:/partlist.png</iconset>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>10</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement" >
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7" >
      <item>
       <widget class="PartsWidget" native="1" name="paintWidget" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QFormLayout" name="formLayout" >
       <property name="fieldGrowthPolicy" >
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="labelAlignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="sizeIcon" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="descrLabel" >
           <property name="text" >
            <string>size:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="sizeLabel" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QLabel" name="readyIcon" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="descrLabel_2" >
           <property name="text" >
            <string>ready:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="readyLabel" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QLabel" name="availableIcon" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="descrLabel_3" >
           <property name="text" >
            <string>available:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="availableLabel" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <widget class="QLabel" name="rareIcon" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="descrLabel_4" >
           <property name="text" >
            <string>rare:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1" >
        <widget class="QLabel" name="rareLabel" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <widget class="QLabel" name="missingIcon" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="descrLabel_5" >
           <property name="text" >
            <string>missing:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1" >
        <widget class="QLabel" name="missingLabel" >
         <property name="text" >
          <string>TextLabel</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PartsWidget</class>
   <extends>QWidget</extends>
   <header>partswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>partListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>442</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel" >
     <x>268</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>partListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>442</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel" >
     <x>268</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
